From a5e29f8520ba2dc65af47153980ec3401e18757b Mon Sep 17 00:00:00 2001
From: Adrian Schmutzler <freifunk@adrianschmutzler.de>
Date: Mon, 30 Sep 2019 17:09:10 +0200
Subject: [PATCH 1/2] babeld: Include PKG_RELEASE in babeld version

This will account for custom patches added, as otherwise version
would stay the same.

Signed-off-by: Adrian Schmutzler <freifunk@adrianschmutzler.de>
---
 babeld/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/babeld/Makefile b/babeld/Makefile
index 022d0b8..cc266ce 100644
--- a/babeld/Makefile
+++ b/babeld/Makefile
@@ -48,6 +48,11 @@ MAKE_FLAGS+= \
 	CFLAGS="$(TARGET_CFLAGS)" \
 	LDLIBS="" \
 
+define Build/Configure
+	echo "openwrt-$(PKG_VERSION)-$(PKG_RELEASE)" > $(PKG_BUILD_DIR)/version
+	$(Build/Configure/Default)
+endef
+
 define Package/babeld/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/babeld $(1)/usr/sbin/
-- 
2.20.1

